ide/coqide.ml

```diff
@@ -888,7 +888,7 @@ let alpha_items menu_name item_name l =
     | [s] -> mk_item s
     | s::_ as ll ->
       let name = item_name^" "^(String.make 1 s.[0]) in
```

Comment:
Using `sprintf "%s %c" item_name s.[0]` would be (more efficient and) more harmonious/consistent with the new definition of `label` below.